src/js/main.js

```diff
@@ -38,6 +42,10 @@ Site = {
       _this.Index.getHeaderTop();
     }
 
+    if ($('.blog-item, .blog-item-mobile, .blog-item-single').find('iframe').length) {
```

Comment:
better to reference the `Site.Blog.$blogIframes` var here I think. Otherwise this is going to fire a lot on a normal non debounced window resize